pynsist.cfg

```diff
@@ -42,18 +51,64 @@ pypi_wheels = PyQt5==5.14.2
     pywin32==227
     send2trash==1.7.1
 files = share/
-exclude = pkgs/PyQt5/QtBluetooth.pyd
-    pkgs/PyQt5/QtDesigner.pyd
-    pkgs/PyQt5/QtQuick.pyd
-    pkgs/PyQt5/QtQuickWidgets.pyd
-    pkgs/PyQt5/QtQml.pyd
-    pkgs/PyQt5/Qt/bin/Qt5Bluetooth.dll
+exclude = pkgs/PyQt5/Qt/bin/Qt5Bluetooth.dll
+    pkgs/PyQt5/Qt/bin/Qt5DBus.dll
     pkgs/PyQt5/Qt/bin/Qt5Designer.dll
+    pkgs/PyQt5/Qt/bin/Qt5Help.dll
+    pkgs/PyQt5/Qt/bin/Qt5Location.dll
+    pkgs/PyQt5/Qt/bin/Qt5Multimedia.dll
+    pkgs/PyQt5/Qt/bin/Qt5MultimediaWidgets.dll
+    pkgs/PyQt5/Qt/bin/Qt5NetworkAuth.dll
+    pkgs/PyQt5/Qt/bin/Qt5Nfc.dll
+    pkgs/PyQt5/Qt/bin/Qt5Positioning.dll
+    pkgs/PyQt5/Qt/bin/Qt5PositioningQuick.dll
+    pkgs/PyQt5/Qt/bin/Qt5PrintSupport.dll
+    pkgs/PyQt5/Qt/bin/Qt5Qml.dll
+    pkgs/PyQt5/Qt/bin/Qt5QmlModels.dll
+    pkgs/PyQt5/Qt/bin/Qt5QmlWorkerScript.dll
     pkgs/PyQt5/Qt/bin/Qt5Quick.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickControls2.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickParticles.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickShapes.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickTemplates.dll
+    pkgs/PyQt5/Qt/bin/Qt5QuickTemplates2.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickTest.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickWidgets.dll
-    pkgs/PyQt5/Qt/bin/Qt5Qml.dll
+    pkgs/PyQt5/Qt/bin/Qt5RemoteObjects.dll
+    pkgs/PyQt5/Qt/bin/Qt5Sensors.dll
+    pkgs/PyQt5/Qt/bin/Qt5SerialPort.dll
+    pkgs/PyQt5/Qt/bin/Qt5Sql.dll
+    pkgs/PyQt5/Qt/bin/Qt5Test.dll
+    pkgs/PyQt5/Qt/bin/Qt5WebSockets.dll
+    pkgs/PyQt5/Qt/bin/Qt5WinExtras.dll
+    pkgs/PyQt5/Qt/bin/Qt5Xml.dll
+    pkgs/PyQt5/Qt/bin/Qt5XmlPatterns.dll
+    pkgs/PyQt5/Qt/bin/concrt140.dll
+    pkgs/PyQt5/Qt/bin/d3dcompiler_47.dll
+    pkgs/PyQt5/Qt/bin/libeay32.dll
+    pkgs/PyQt5/Qt/bin/msvcp140.dll
+    pkgs/PyQt5/Qt/bin/vcruntime140.dll
+    pkgs/PyQt5/Qt/plugins/audio
+    pkgs/PyQt5/Qt/plugins/generic
+    pkgs/PyQt5/Qt/plugins/geometryloaders
+    pkgs/PyQt5/Qt/plugins/geoservices
+    pkgs/PyQt5/Qt/plugins/mediaservice
+    pkgs/PyQt5/Qt/plugins/platforms/qminimal.dll
+    pkgs/PyQt5/Qt/plugins/platforms/qoffscreen.dll
+    pkgs/PyQt5/Qt/plugins/platforms/qwebgl.dll
+    pkgs/PyQt5/Qt/plugins/platformthemes
+    pkgs/PyQt5/Qt/plugins/playlistformats
+    pkgs/PyQt5/Qt/plugins/position
+    pkgs/PyQt5/Qt/plugins/printsupport
+    pkgs/PyQt5/Qt/plugins/sceneparsers
+    pkgs/PyQt5/Qt/plugins/sensorgestures
+    pkgs/PyQt5/Qt/plugins/sensors
+    pkgs/PyQt5/Qt/plugins/sqldrivers
+    pkgs/PyQt5/Qt/plugins/styles
+    pkgs/PyQt5/Qt/plugins/texttospeech
+    pkgs/PyQt5/Qt/plugins/webview
+    pkgs/PyQt5/QtBluetooth.pyd
+    pkgs/PyQt5/QtDesigner.pyd
+    pkgs/PyQt5/QtQml.pyd
+    pkgs/PyQt5/QtQuick.pyd
+    pkgs/PyQt5/QtQuickWidgets.pyd
```

Comment:
Thanks again and sorry for taking so long to get back to this. I may actually go ahead and remove opengl post-merge (don't worry about it) since that's a hefty one ~ thanks for the note about the sizes. If something breaks we can always add it later.

One last request, though. Please resave this file because and amend the commit after adjusting your editor settings. This file is missing the terminating newline.

https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Sure! I have added the terminating newline and also added the `opengl32sw.dll` file in exclusion list.

Reply:
Oh wait, the newline is gone after commit. Git is probably removing it, I'll update it.

Reply:
Ok I had some confusion but I think now it is done, can you see if the newline is added?